DQMServices/Core/python/nanoDQMIO_perLSoutput_cff.py

```diff
@@ -139,16 +139,16 @@
     "SiStrip/MechanicalView/MainDiagonal Position",
     "SiStrip/MechanicalView/NumberOfClustersInPixel",
     "SiStrip/MechanicalView/NumberOfClustersInStrip",
-    "Tracking/TrackParameters/GeneralProperties/Chi2oNDF_lumiFlag_GenTk",
-    "Tracking/TrackParameters/GeneralProperties/NumberOfRecHitsPerTrack_lumiFlag_GenTk",
-    "Tracking/TrackParameters/GeneralProperties/NumberOfTracks_lumiFlag_GenTk",
+    "Tracking/TrackParameters/GeneralProperties/Chi2oNDF_lumiFlag_GenTk",  #non existing
```

Comment:
I think you might want instead this: 

```suggestion
"Tracking/TrackParameters/generalTracks/LSanalysis/Chi2oNDF_lumiFlag_GenTk",
```

Similarly for the two other ones.  The current string looks like it's for the online DQM (but not quite...) 
@gbenelli FYI